pkg/executor/batch_point_get.go

```diff
@@ -534,9 +544,18 @@ type cacheBatchGetter struct {
 	snapshot kv.Snapshot
 }
 
-func (b *cacheBatchGetter) BatchGet(ctx context.Context, keys []kv.Key) (map[string][]byte, error) {
+func (b *cacheBatchGetter) BatchGet(ctx context.Context, keys []kv.Key, options ...kv.BatchGetOption) (map[string]kv.ValueEntry, error) {
+	var getOptions tikv.BatchGetOptions
+	if len(options) > 0 {
+		getOptions.Apply(options)
+	}
+
+	if getOptions.RequireCommitTS() {
+		return nil, errors.New("WithReturnCommitTS option is not supported for pessimistic lock cache getter")
```

Comment:
there are both RequireCommitTS and WithReturnCommitTS

Reply:
RequireCommitTS is from client-go, it should be updated in another PR